sdk/monitor/monitor-ingestion/samples-dev/earlyAborting.ts

```diff
@@ -10,21 +10,20 @@ import { DefaultAzureCredential } from "@azure/identity";
 import {
   isAggregateLogsUploadError,
   LogsIngestionClient,
-  LogsUploadFailure,
+  type LogsUploadFailure,
 } from "@azure/monitor-ingestion";
+import "dotnev/config";
```

Comment:
seems like same typo as above?
```suggestion
import "dotenv/config";
```